package.json

```diff
@@ -77,8 +77,10 @@
     "stylelint-scss": "^1.4.3",
     "stylelint-webpack-plugin": "^0.9.0",
     "through2": "^2.0.3",
+    "url-loader": "^0.6.2",
     "webpack": "^3.8.1",
     "webpack-dev-server": "^2.9.4",
+    "webpack-iconfont-plugin": "0.0.2",
```

Comment:
Per some other comment somewhere, do these new icon processing libraries have a dependency on the fontconfig server package or other packages?